editor-components/stats/stats7/stats7.tsx

```diff
@@ -107,34 +123,66 @@ class Stats7Page extends BaseStats {
   }
 
   render() {
-    const styling = {
-      backgroundColor: this.getPropValue("backgroundColor"),
-    };
+    const subtitle = this.getPropValue("subtitle")
+    const title = this.getPropValue("title")
+    const description = this.getPropValue("description")
+  
+
 
     return (
       <div className={this.decorateCSS("container")} >
-        <div className={this.decorateCSS("max-content")} style={styling}>
-          <div className={this.decorateCSS("stats1-page")}>
-            <div className={this.decorateCSS("title-child")}>
-              <h1 className={this.decorateCSS("title")}>{this.getPropValue("title")}</h1>
-              <h3 className={this.decorateCSS("description")}>{this.getPropValue("description")}</h3>
-            </div>
-            <div className={this.decorateCSS("bottom-child")}>
-              {this.castToObject<ICard[]>("card-content").map(
-                (cardData: any, indexCard: number) => (
-                  <div key={`sts-7-${indexCard}`} className={this.decorateCSS("card")}>
-                    <h5 className={this.decorateCSS("card-data-subtitle")}>{cardData.subtitle}</h5>
-                    <h4 className={this.decorateCSS("card-data-title")}>{cardData.title}</h4>
-                    <p className={this.decorateCSS("card-data-description")}>{cardData.description}</p>
+        <div className={this.decorateCSS("max-content")}>
+
+          {subtitle && <div className={this.decorateCSS("title-child")}>
+            {subtitle && (
+              <h1 className={this.decorateCSS("subtitle")}>{subtitle}</h1>
+            )}
+            {title && (
+              <h1 className={this.decorateCSS("title")}>{title}</h1>
+            )}
+            {description && (
+              <p className={this.decorateCSS("description")}>
+                {description}
+              </p>
+            )}
+          </div>}
+
+          <div className={this.decorateCSS("progress-container")}>
+            {this.getPropValue("items").map((item: any, index: number) => {
```

Comment:
please cast to object. use the type you wrote.
```suggestion
            {this.castToObject<Item[]>("items").map((item: Item, index: number) => {
```